internal/models/producto.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+type TipoIngrediente string
```

Comment:
¿Has considerado plantear este type de otra forma.? Lo digo porque aunque sigamos el SSOT (single source of truth). Hay alimentos perecederos y otros no. Tiene que haber alguna forma de ditinguirlos. ¿La sal o el agua se considera un alimento de tipo perecedero?

Reply:
> ¿La sal o el agua se considera un alimento de tipo perecedero?
No son alimentos perecederos, pero si lo vuelvo a poner como perecederos y no perecederos volvemos a lo mismo de que se puden guardar alimentos con mismo nombre, pero diferente tipo, que ahora mismo tambien puede pasar, pero con tipo perecedero o null.
Si hago alguna funcion para comprobar el tipo o que el nombre no se repita no sería un PMV.

Reply:
He añadido peredecero y no perecedero para poder diferenciar el tipo de Ingrediente. 
Para mantener SSOT el mapa de `Inventario` evita que existan dos `Ingredientes` con el mismo `nombre`, por la tanto ya no pueden haver varios `Ingredientes` con diferentes tipos.
`Ingrediente` sigue siendo la entidad y `Producto` el objeto valor.

Reply:
Perfecto y muchas gracias